vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/ImageServiceTest.java

```diff
@@ -150,6 +150,38 @@ public void test_getImages_pageGreaterThanTotalPages() {
         verify(imageRepo).findAll(sortByRequestedField);
     }
     
+    @Test
+    public void test_getImages_sorted_success() { 
+        Pageable sortByRequestedField = PageRequest.of(0, 10, Sort.by(SortByField.CREATION_DATE.getValue()).descending());
+        when(imageRepo.count()).thenReturn(1L);
+        when(imageRepo.findAll(sortByRequestedField)).thenReturn(new PageImpl<VSImage>(images));
+        List<IVSImage> requestedImages = serviceToTest.getImages(1, SortByField.CREATION_DATE.getValue(), Sort.Direction.DESC.toString());
+        assertEquals(1, requestedImages.size());
+        assertEquals(IMG_ID, requestedImages.get(0).getId());
+        verify(imageRepo).findAll(sortByRequestedField);
```

Comment:
since you have a return value to test if the method does what it's supposed, I wouldn't verify this. There could be other ways of getting the correct results that do not call `findAll` and by putting this here, your test case will break if we change the implementation.